app/controllers/miq_policy_controller/alerts.rb

```diff
@@ -644,7 +647,7 @@ def alert_get_info(alert)
       @alert_profiles = @alert.memberof.sort_by { |p| p.description.downcase }
     end
 
-    unless @alert.expression.kind_of?(MiqExpression) # Get the EMS if it's in the expression
+    unless @alert.expression.nil? || @alert.expression.kind_of?(MiqExpression) # Get the EMS if it's in the expression
```

Comment:
using `unless` with more than one condition is confusing imo. 
also, what kind of expression/reason for the second added condition?